src/renderer/src/lib/i18n/locales/ja.json

```
@@ -951,6 +952,244 @@
+        "input_error": {
+          "already_touch": {
+            "title": "すでに Touch Mod です",
+            "description": "生成済みの Touch Mod を再変換することはできません。元の non-touch Mod を選択するか、既存の変換をロールバックしてから再試行してください。"
+          },
+          "suspected_touch": {
+            "title": "Touch Mod の可能性があります",
+            "description": "既存の Touch ランタイムが検出されたため、安全に変換できません。元の non-touch Mod を選択してください。"
+          }
+        },
+        "skipped": "除外",
+        "preview_title": "タッチ影響範囲プレビュー",
```

Biome (2.5.6)

[error] The key preview_title was already declared.

(lint/suspicious/noDuplicateObjectKeys)